### ISSUE TITLE
Blank window pops up when loading Rlabkey in Rstudio (windows)

### ISSUE THREAD

**Issue by juyeongkim** [LabKey/labkey-api-r#14]
When loading the `Rlabkey` package in Rstudio on an windows machine, a blank window pops up instead of the user guide vignette (as it was intended on [schemaObjects.R#L456](https://github.com/LabKey/labkey-api-r/blob/develop/Rlabkey/R/schemaObjects.R#L456)).

```R
> library(Rlabkey)
Loading required package: RCurl
Loading required package: bitops
Loading required package: rjson
> sessionInfo()
R version 3.4.1 (2017-06-30)
Platform: i386-w64-mingw32/i386 (32-bit)
Running under: Windows 7 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets 
[6] methods   base     

other attached packages:
[1] Rlabkey_2.1.135 rjson_0.2.15    RCurl_1.95-4.8 
[4] bitops_1.0-6   

loaded via a namespace (and not attached):
[1] compiler_3.4.1 tools_3.4.1    Rcpp_0.12.12  
```

![screenshot_20170802_093427](https://user-images.githubusercontent.com/5667572/28884413-93503b06-7766-11e7-8965-a1c878fecbc6.png)

`Rstudio version: 1.0.153`

Could you look into this?

Also, I was wondering if showing the vignette is even necessary when loading the package. It could be bothersome for some users who have been using the package for a while and don't need to see the user guide every time they load the package.

**Comment by cnathe** [comment]
Thank you for the feedback. I have seen that blank window open as well locally. I'll take a look into this and hopefully get an updated Rlabkey package with a fix posted soon.

**Comment by cnathe** [comment]
This should be fixed in the latest Rlabkey version on CRAN 2.1.136
